cypress/support/commands.js

```diff
@@ -181,6 +182,8 @@ Cypress.Commands.add("addWallet", (walletName, walletType, funded, nodeType, key
       }
       cy.get('[data-cy="create-wallet-btn"]').click()
       cy.get('[data-cy="new-wallet-added-headline"]')
+      cy.get('#pdf-wallet-download').click()
```

Comment:
I don't like that we check this here in the general command! Slows down the tests, makes them super vulnerable. Why not test this once in a dedicated test?